oneview_redfish_toolkit/mockups/oneview/ServerProfileWithLabels.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The name of this file ("ServerProfileWithLabels") is not representing the correct resource, because the resource is not a ServerProfile, it is a `"type":"ResourceLabels"`. Maybe we could name it as "LabelForServerProfile".
What do you think?

Reply:
Seems a great suggestion. I will rename it.

Reply:
Rename the variable names representing it mockup, too. =)